embedding/cpp/include/elm_flutter_view.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_ELM_FLUTTER_VIEW_H_
+#define FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_ELM_FLUTTER_VIEW_H_
+
+#include <Elementary.h>
+#include <flutter/plugin_registry.h>
+#include <flutter_tizen.h>
+
+#include <string>
+#include <vector>
+
+// The app base class which creates and manages the Flutter engine instance.
+class ElmFlutterView : public flutter::PluginRegistry {
+ public:
+ explicit ElmFlutterView() {}
```

Comment:
The tab size must be 2.
```suggestion
  explicit ElmFlutterView() {}
```

Reply:
thanks!